### ISSUE TITLE
Index as First page?

### ISSUE THREAD

**Issue by db-dzine** [mpdf/mpdf#423]
### These are mPDF and PHP versions I am using 6.1.

**Comment by finwe** [comment]
No, not ATM, and it is not even planned. Marking as feature request.

**Comment by wimurk** [comment]
+1

**Comment by darkavatar23** [comment]
+1

**Comment by SmileYzn** [comment]
To some users like me that wants to do this:

For now, you can create a pdf document and import it using last page of created document into desired page that will have a index

`
$mpdf = new \Mpdf\Mpdf();

$paginas = $mpdf->SetSourceFile('temp.pdf');
$indice  = $mpdf->importPage($paginas);

for($pagina = 1;$pagina < $paginas;$pagina++)
{
    $mpdf->AddPage();
    
    $tp = $mpdf->importPage($pagina);
    $mpdf->useTemplate(($pagina == 2) ? $indice : $tp);
}

$mpdf->Output();`

Ps. $pagina is current page, also i ignored last page in for loop duo last pasge is index itself.

I hope i can help.
